addons/website/data/website_data.xml

```diff
@@ -232,6 +232,7 @@
             <field name="domain"></field>
             <field name="company_id" ref="base.main_company"/>
             <field name="user_id" ref="base.public_user"/>
+            <field name="auth_signup_uninvited">b2c</field>
```

Comment:
From commit message

> Free sign up is the default option for the default website's authentication setting (auth_signup_uninvited).

Can you remind me where this default b2c comes from before this PR?

Reply:
In odoo/addons/auth_signup/data/auth_signup_data.xml, there is this line:
`
<!-- activate B2C portal by default -->
<function model="ir.config_parameter" name="set_param" eval="('auth_signup.invitation_scope', 'b2c')"/>
`
Which activates the B2C portal by default when installing tha auth_signup addon, but since that's not the value that is taken into account when it becomes website specific (website's own value overrides it, and it's b2b by default), it needs to be set correctly on install through the website data file IMO.

Reply:
> So it never was B2C by default for My Website, it was only visually the case

Ok that's what I thought and asked the question.
Then this change (b2c default) is not suitable for a stable version, it would be a behavior change, after X years people creating a website in the same stable version will suddenly have a B2C website where people can register.

So, without this, the option will visually be fixed and set to B2B, as it will correctly read from the website & not the ICP. Which is correct and enough.

For master, maybe @pierre-pde can tell you what he wants -> free signup or not by default for website?
Note that if he wants B2C, you should just change the `default` of the website field declaration

Reply:
Remove this, we will keep B2B, you don't want people to register on your website 'vitrine'